app/components/view-goal/progress-bar.jsx

```diff
@@ -1,31 +1,32 @@
 import React from 'react';
 
-const ProgressBarComponent = ({ progress, time, }) => {
+const ProgressBarComponent = ({ rating, time, }) => {
+  const score = parseInt(rating, 10);
```

Comment:
Parse this as it comes out of the event, so it is an int everywhere in your application. Good work with the second argument